src/pages/Data/BorrowData.tsx

```diff
@@ -21,6 +21,7 @@ import { ErrorBox } from '../../components/ErrorBox';
 import { Modal } from '../../components/Modal';
 import { useGlobalStore } from '../../store';
 import { Spinner } from '../../components/Spinner';
+import { logger } from '../../utils/logger';
 
 interface BorrowEligibilityResp {
```

Comment:
You already created and exported this interface in `src/utils/Interface.ts` consider importing instead.

Reply:
I've updated this too